### PR TITLE
Improves queue inspection tool to show progress and have a settable cache size

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

**Comment by chtyim** [comment]
Build: http://builds.cask.co/browse/CDAP-RBT284-1

File: cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java [hunk 1]
```diff
@@ -166,7 +166,7 @@ private void scanQueue(TransactionExecutor txExecutor, HBaseConsumerStateStore s
 
     HBaseQueueAdmin admin = queueClientFactory.getQueueAdmin();
     TableId tableId = admin.getDataTableId(queueName, QueueConstants.QueueType.SHARDED_QUEUE);
-    HTable hTable = queueClientFactory.createHTable(tableId);
+    final HTable hTable = queueClientFactory.createHTable(tableId);
```
Comment (jwang47) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java]: Why final?

**Comment by jwang47** [comment]
One minor comment, otherwise LGTM.

File: cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java (hunk 1 above)
Comment (chtyim) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java]: Added by IDE when I move code around. Removing it.

**Comment by jwang47** [comment]
LGTM

**Comment by jwang47** [comment]
LGTM

**Comment by jwang47** [comment]
LGTM

**Comment by chtyim** [comment]
Squashed commit and merging.

**Pull request merged**